### ISSUE TITLE
Admin secret is not usable after deploying new awx version with unmanaged/external postgresql db

### ISSUE THREAD

**Issue by A-Kumpf** [ansible/awx#13728]
##### ISSUE TYPE

 - Breaking Change

##### COMPONENT NAME

 - Login

##### SUMMARY

Salü
After deploying the new awx version 21.13.0 with an unmanaged/external postgresql db, I'm not able to login with admin secret (invalid user/password).
Only after creating a new superuser with password login works.
Doing the same deployment with a managed db, I'm able to login with the the admin secret.
Regards

##### ENVIRONMENT
* AWX version: 21.13.0
* AWX install method: operator
* AWX deployment target: kubernetes
* Operating System: Linux
* Web Browser: Firefox

##### STEPS TO REPRODUCE

Deployment as described in README.md with externally managed postgresql.

##### EXPECTED RESULTS

Login works as with a managed db.

##### ACTUAL RESULTS

Login doesn't work.

**Comment by rooftopcellist** [comment]
@A-Kumpf This is more of an awx-operator issue FYI, it might be more useful to open an issue there.  

Here are the tasks that create the admin user:
* https://github.com/ansible/awx-operator/blob/devel/roles/installer/tasks/initialize_django.yml#L2-L48

Is your database empty? or did you populate it with a pg_dump from an existing AWX and pg_restore to the database?

If so, there is probably an admin user in the database already, so the check task in the playbook I linked would make it so that the following task skips attempting to create an admin user.  

------------------
As a work-around, you could run `awx-manage createsuperuser` as you mentioned, or modify the `admin_user` on the AWX CR spec and it will create a user by a different name.

**Comment by djyasin** [comment]
We are going to go ahead and close this issue due to inactivity. If you are still encountering trouble, please open a new issue and reference this one. Thank you for your time!
